lib/surface/compiler.ex

```diff
@@ -591,6 +591,7 @@ defmodule Surface.Compiler do
 
   defp collect_templates(mod, attributes, nodes, meta) do
     # Don't extract the template directives if this module is slotable
+    # TODO: and renderless ?
```

Comment:
this TODO is because I wasn't sure if we needed to filter out renderable slotable components here. I don't /think/ we need to change this, but I could be missing something